scrapers/govScrapers/getVietnam.js

```diff
@@ -0,0 +1,38 @@
+const axios = require('axios');
+const cheerio = require('cheerio');
+const logger = require('../../utils/logger');
+
+const columns = ['city', 'cases', 'beingTreated', 'recovered', 'deaths'];
+
+const mapRows = (_, row) => {
+	const city = { updated: Date.now() };
+	cheerio(row).children('td').each((index, cell) => {
+		cell = cheerio.load(cell);
+		switch (index) {
+			case 0: {
+				city[columns[index]] = cell.text().trim();
+				break;
+			}
+			default: {
+				city[columns[index]] = parseInt(cell.text().replace(/\./g, ''));
+				break;
+			}
+		}
+	});
+	return city;
+};
+
+/**
+ * Scrapes Vietnam government site and fills array of data from table
+ */
+const vietnamData = async () => {
+	try {
+		const html = cheerio.load((await axios.default({ method: 'GET', url: 'http://ncov.moh.gov.vn/', httpsAgent: new (require('https').Agent)({ rejectUnauthorized: false }) })).data);
```

Comment:
For anyone interested, we are setting rejectUnauthorized to false here so we can request the vietnamese government site without getting SSL errors because their SSL chain is invalid. Most browsers fill that chain themselves but axios does not do that of course.